package.json

```diff
@@ -22,21 +22,28 @@
     "lib/",
     "src/"
   ],
+  "type": "module",
   "source": "src/eventmit.ts",
-  "main": "dist/eventmit.js",
-  "types": "dist/eventmit.d.ts",
-  "module": "dist/eventmit.module.js",
-  "esmodule": "dist/eventmit.modern.js",
-  "unpkg": "dist/eventmit.umd.js",
+  "main": "cjs/eventmit.js",
+  "types": "module/eventmit.d.ts",
+  "module": "module/eventmit.js",
+  "exports": {
+    ".": {
+      "import": "./module/eventmit.js",
+      "require": "./cjs/eventmit.js",
+      "default": "./cjs/eventmit.js"
+    },
+    "./package.json": "./package.json"
+  },
   "directories": {
     "lib": "lib",
     "test": "test"
   },
   "scripts": {
-    "build": "microbundle",
+    "build": "tsc -p . && tsc -p ./tsconfig.cjs.json && tsconfig-to-dual-package",
```

Comment:
build tsc and [tsconfig-to-dual-package](https://github.com/azu/tsconfig-to-dual-package) add package.json into each outDir